app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+# Create Post model
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
```

Comment:
- [ ] As I mentioned in the previous comment, the author does not exist in your database schema. Please correct the association between the Post and User models.